### ISSUE TITLE
 Cannot resolve 'react/lib/ReactTestUtils'

### ISSUE THREAD

**Issue by MilllerTime** [algolia/expect-jsx#25]
Hey there!

I've been using expect-jsx for several months, but now when compiling my tests with webpack I'm getting the following error:
```
Module not found: Error: Cannot resolve module 'react/lib/ReactTestUtils'
```

Webpack also throws a runtime error, but the only useful information in the stack trace is that the error starts on the line where I import expect-jsx.

I just updated React to 15.4.1, and I'm still on expect-jsx 2.6.0. This issue started after the React update. I tried rebuilding `node_modules` without resolution.

It appears expect-jsx is relying on an outdated version of react-element-to-jsx-string (3.0.0). Is it possible this outdated dependency is importing react test utils in a manner that breaks on the current version of React?

Thanks in advance for any insight!

**Comment by vvo** [comment]
Should be fixed in 3.0.0, let me know

**Comment by MilllerTime** [comment]
That worked! Thanks for the update 👍
